conventions.md

```diff
@@ -0,0 +1,301 @@
+                        durable: true
+                        autoDelete: false
+                        bindingVersion: 0.1.0
+    bindings:
+        amqp:
+            exchange:
+                name: myExchange
+                type: topic
+                durable: true
+                autoDelete: false
+                vhost: /
+                bindingVersion: 0.1.0
+```
+
+```yml
+channels:
+    quux:
+        publish:
+            summary: Messages about quux
+            operationId: sendQuux
+            $ref: "#/components/messages/quuxEvent"
+            bindings:
+                kafka:
+                    consumer:
+                        group: myGroup
+                        maxPollIntervalInMs: 100
+                        enableAutoOffsetStore: true
+                        enableAutoCommit: true
+    bindings:
+        kafka:
+            topic:
+                partitioner: ConsistentRandom
+                numOfPartitions: 6
+                replicationFactor: 3
+```
+
+
+Note that the consumer defines what it exposes - a publish operation because it can be published to - so the middleware 'subscription' is defined on the consumer's publish operation. This catches out some users of AsyncAPI.
+
+### **Item 9** Define Protocol Required Message Properties as Schema not Bindings
+A [Message](https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md#messageBindingsObject) consists of headers (metadata) and payload (data); AsyncAPI also defines some common metadata values as fields of a Message, for example correlationId. The headers are defined as a [Schema](https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md#schemaObject) object under [Components](https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md#componentsObject). A payload may be of any type, but defaults to JSON Schema. [MessageTraits](https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md#messageTraitObject) allow the definition of metadata values related to a protocol once, rather than per message, and simplify the use of protocol related message metadata when defining message headers.
+
+Where a protocol defines metadata that forms part of the message sent via middleware it should be defined as part of the header schema (potentially as MessageTraits), and where the protocol defines data (or transmits metadata in the payload) it should be defined as part of the payload. Note that the payload does not support MessageTraits. 
+
+Tooling to validate that messages match the schema will expect to verify against the schema of the message payload and headers, putting message metadata into a Message Binding makes this harder as tooling must know to look there as well. 
+
+Implementers of producers or consumers are also likely to assume that the the can refer to #/Components/Schema to understand payloads and headers, and adding metadata into bindings in addition makes it harder to comprehend the schema of a message.
```

Comment:
This is not true. The right place to set protocol-specific message properties is actually the bindings, not traits or headers. See the definition of the [Message headers field](https://www.asyncapi.com/docs/reference/specification/v2.6.0#messageObject):

> Schema definition of the application headers. Schema MUST be of type "object". **It MUST NOT define the protocol headers**.

The reason this is separated is to prevent mixing protocol-specific and application-specific metadata in a single place and the user having to define the shape of the metadata with JSON Schema every time.

My suggestion is that we remove the whole Item 9.